ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/techpreview/provision/cucushift-installer-rehearse-gcp-ipi-xpn-techpreview-provision-chain.yaml

```diff
@@ -0,0 +1,32 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-techpreview-provision
+  steps: 
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-credentials-mode
+    - ref: ipi-conf-feature-set
+    - ref: ipi-conf-gcp-xpn-network
+    - ref: ipi-conf-gcp-create-firewall-rules
+    - ref: ipi-conf-gcp-public-dns-zone
+    - chain: ipi-install
```

Comment:
If I remember right, after the install get completed, we need to add a public dns record for "*.apps", right?

Reply:
That's should be done during IPI installation (e.g. by ingress operator), rather than manually add it.

Reply:
So now ingress operator will add it during IPI installation?

Reply:
No, I mean it should but currently it doesn't do that.

Reply:
Once we added the workflow into ci, many e2e test need to access "*.apps", to ensure the testing move on, we need to add a separate step to add the DNS record as workaround, once the issue is fixed, we can remove the workflow later.

Reply:
1. Do you mean only add the DNS record in the public DNS zone? 
2. Do you mean when there's no platform.gcp.privateDNSZone specified?

Reply:
I meant 1, for 1, maybe need to develop a new step.